src/program/l2fwd/l2fwd.lua

```diff
@@ -0,0 +1,70 @@
+module(...,package.seeall)
+
+local lib = require("core.lib")
+local main = require("core.main")
+
+L2Fwd = {}
+
+function L2Fwd.new(conf)
+   local o = {}
+   return setmetatable(o, { __index = L2Fwd })
+end
+
+function L2Fwd:push()
+   local input, output = assert(self.input.input), assert(self.output.output)
+
+   while not link.empty(input) and not link.full(output) do
```

Comment:
Following https://github.com/SnabbCo/snabbswitch/pull/664#issuecomment-160889463, let's remove the `and not link.full(output)` check.